.docker/nginx/browser.conf

```diff
@@ -17,8 +17,8 @@ server {
         fastcgi_pass app:9000;
         fastcgi_split_path_info ^(.+\.php)(/.*)$;
 
-        fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
-        fastcgi_param DOCUMENT_ROOT $realpath_root;
+        fastcgi_param SCRIPT_FILENAME /app/public/$fastcgi_script_name;
```

Comment:
doesn't match `root`?

Reply:
I see the folders are mounted in different places in the `nginx` and the other app containers, though this shows how to make the paths different if needed, it could be easier to keep everything consistent for maintenance

Reply:
Now supporting this, with named volumes in `sample-configuration`